README.md

```diff
@@ -0,0 +1,32 @@
+OregonDigital\_2
+===============
+
+[![Circle CI](https://circleci.com/gh/OregonDigital/oregondigital_2.svg?style=svg)](https://circleci.com/gh/OregonDigital/oregondigital\_2)
+
+
+Local Development Setup
+-----
+
+Requirements:
+
+  **Ruby Version: 2.0**
+
+Configuration and database creation:
+
+    git clone git@github.com:OregonDigital/oregondigital_2.git
+	cd oregondigital_2
```

Comment:
I don't know if you wanted the double-indent to show up here, but github hides it... I'm not sure if there's any way to make it show up.